db/migrate/20160322205357_remove_host_id_from_switch.rb

```diff
@@ -0,0 +1,41 @@
+class RemoveHostIdFromSwitch < ActiveRecord::Migration[5.0]
+  class Host < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    has_many :host_switches,
+             :dependent   => :destroy,
+             :class_name  => 'RemoveHostIdFromSwitch::HostSwitch'
+    has_many :switches,
+             :through => :host_switches,
+             :class_name  => 'RemoveHostIdFromSwitch::Switch'
+  end
+
+  class Switch < ActiveRecord::Base
+    has_many :host_switches,
+             :dependent   => :destroy,
+             :class_name  => 'RemoveHostIdFromSwitch::HostSwitch'
+    has_many :hosts,
+             :through => :host_switches,
+             :class_name => 'RemoveHostIdFromSwitch::Host'
+  end
+
+  class HostSwitch < ActiveRecord::Base
+    belongs_to :host,
+               :class_name  => 'RemoveHostIdFromSwitch::Host'
+    belongs_to :switch,
+               :class_name => 'RemoveHostIdFromSwitch::Switch'
+  end
+
+  def up
+    remove_column :switches, :host_id, :bigint
```

Comment:
And that was generated by following [this](http://edgeguides.rubyonrails.org/active_record_migrations.html#creating-a-standalone-migration)